test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRealization/GivenBuildingHasInvalidStatus.cs

```diff
@@ -55,8 +55,8 @@ public void ThenValidationException()
                 .Result
                 .Where(x =>
                     x.Errors.Any(
-                        failure => failure.ErrorCode == "GebouwNietgerealiseerdOfGehistoreerd"
-                                    && failure.ErrorMessage == "Deze actie is enkel toegestaan voor een 'gepland' of 'gerealiseerd' gebouw."));
+                        failure => failure.ErrorCode == "GebouwStatusNietInGeplandOfGerealiseerd"
+                                   && failure.ErrorMessage == "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw."));
```

Comment:
Doesn't there need to be ' between the statusses?

Reply:
Nope. Already checked that with Anaïs.